File: MuneneCalvin/Reverse Integer.py
def reverse_integer(num):
    # Handle negative numbers
    sign = -1 if num < 0 else 1
    num = abs(num)
    
    # Reverse the digits
    reversed_num = 0
    while num > 0:
        reversed_num = reversed_num * 10 + num % 10
        num //= 10
    
    # Apply the original sign and return
    return sign * reversed_num

# Test cases
# test_cases = [500, -56, -90, 91]

# for case in test_cases:
#     result = reverse_integer(case)
#     print(f"Input: {case}, Output: {result}")

# Input from the user
num = int(input("Enter an integer: "))
result = reverse_integer(num)
print(f"Reversed integer: {result}")

File: MuneneCalvin/Reverse Integer.js
function reverseInteger(num) {
    // Determine the sign
    const sign = num < 0 ? -1 : 1;
    
    // Convert to positive number and then to string
    let numStr = Math.abs(num).toString();
    
    // Reverse the string
    let reversedStr = numStr.split('').reverse().join('');
    
    // Convert back to number and apply the sign
    return sign * parseInt(reversedStr, 10);
}

// // Test cases
// const testCases = [500, -56, -90, 91];

// testCases.forEach(testCase => {
//     const result = reverseInteger(testCase);
//     console.log(`Input: ${testCase}, Output: ${result}`);
// });

// Input from User
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question('Enter a number: ', num => {
    const result = reverseInteger(parseInt(num, 10));
    console.log(`Reversed number: ${result}`);
    readline.close();
});